neofs-local/.env

```diff
@@ -12,10 +12,10 @@ NEOFS_LOGGER_NO_DISCLAIMER=true
 NEOFS_WORKERS_INDEXER_DISABLED=true
 
 # # Indexer settings
-# NEOFS_WORKERS_INDEXER_DISABLED=false
-# NEOFS_INDEXER_START_FROM=4000
-# NEOFS_INDEXER_ENDPOINT=http://privnet:30333
-# NEOFS_INDEXER_ADDRESSES_0=5f67fd9f24343804837866825c18c33dbbdf4eaf
+#NEOFS_WORKERS_INDEXER_DISABLED=false
+#NEOFS_INDEXER_START_FROM=4000
+#NEOFS_INDEXER_ENDPOINT=http://privnet:30333
+#NEOFS_INDEXER_ADDRESSES_0=5f67fd9f24343804837866825c18c33dbbdf4eaf
```

Comment:
should be `dcc5902c9e8c63286894015ffd27097fd0ac9656`

\+ add comment: `NeoFS smart contract scripthash`